tests/device_tests/test_msg_getownershipproof.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import btc, messages
+from trezorlib.exceptions import TrezorFailure
+from trezorlib.tools import parse_path
+
+
+@pytest.mark.skip_t1
+class TestMsgGetOwnershipProof:
```

Comment:
please remove the class and move testcases to top-level

if you want to mark the whole file, set `pytestmark = pytest.mark.skip_t1`

Reply:
if you want to add `skip_ui`, it's `pytestmark = [pytest.mark.skip_t1, pytest.mark.skip_ui]`

Reply:
b9093c3bc9890fb488737709bb45a145c494329d